ovos_plugin_manager/transformer_services.py

```
@@ -0,0 +1,370 @@
+
+    def feed_hotword(self, chunk: bytes) -> None:
+        """Feed a chunk of hotword audio to all loaded plugins."""
+        for module in self.plugins:
+            module.feed_hotword_chunk(chunk)
+
+    def feed_speech(self, chunk: bytes) -> None:
+        """Feed a chunk of speech audio to all loaded plugins."""
+        try:
+            for module in self.plugins:
+                module.feed_speech_chunk(chunk)
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)